### PR TITLE
Felinids & Vulp Update

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Fixing missing FTL and adding 5 missing DeltaV pr's for Felinids

https://github.com/DeltaV-Station/Delta-v/pull/510
https://github.com/DeltaV-Station/Delta-v/pull/433
https://github.com/DeltaV-Station/Delta-v/pull/553
https://github.com/DeltaV-Station/Delta-v/pull/543
https://github.com/DeltaV-Station/Delta-v/pull/604

Added missing emotes for Vulp:

https://github.com/DeltaV-Station/Delta-v/pull/569

## Why / Balance
Keep with the times.
Bug fixing.

## Technical details
C#
yml

## Media
N/A

## Breaking changes
N/A

**Changelog**
:cl: dvir01
- tweak: Felinids now meow out their words. They can also sigh now.
- add: Added a fluffy tail for the felinids.
- add: Vulpkanins can now Sigh, Sneeze, Cough, Cry and Whistle.

### CODE REVIEW

**Comment by Mnemotechnician** [comment]
So the "tactical deployment kitty turret" is no more. :(

**Comment by FoxxoTrystan** [comment]
Thats nice, also does this fix to put Felinds in bags? like forcing a felind in bag because there the option but does nothing for now.

File: Resources/Locale/en-US/_NF/chat/managers/chat_manager.ftl [hunk 1]
```diff
@@ -1,4 +1,9 @@
 chat-speech-verb-vulpkanin-1 = rawrs
 chat-speech-verb-vulpkanin-2 = barks
 chat-speech-verb-vulpkanin-3 = rurs
-chat-speech-verb-vulpkanin-4 = yeeps
```
Comment (FoxxoTrystan) [inline Resources/Locale/en-US/_NF/chat/managers/chat_manager.ftl:4]: why?

Comment (dvir001) [reply Resources/Locale/en-US/_NF/chat/managers/chat_manager.ftl:4]: DeltaV
Just trying to keep it the same

**Comment by dvir001** [comment]
> Thats nice, also does this fix to put Felinds in bags? like forcing a felind in bag because there the option but does nothing for now.

Didnt test it but it might fix the issue.

File: Resources/Locale/en-US/_NF/chat/managers/chat_manager.ftl (hunk 1 above)
Comment (FoxxoTrystan) [reply Resources/Locale/en-US/_NF/chat/managers/chat_manager.ftl:4]: Il honestly prefer to keep it, just add it as a number five.

Comment (dvir001) [reply Resources/Locale/en-US/_NF/chat/managers/chat_manager.ftl:4]: ok..

**Pull request merged**